middleware/proxy/upstream.go

```diff
@@ -147,6 +160,15 @@ func parseBlock(c *parse.Dispenser, u *staticUpstream) error {
 			return err
 		}
 		u.MaxFails = int32(n)
+	case "max_conns":
```

Comment:
@augustoroman Oops. Submit a PR to https://github.com/caddyserver/caddyserver.com - that would be great! :)